src/lit-html.ts

```diff
@@ -738,7 +744,19 @@ export class TemplateInstance {
           index++;
           walker.nextNode();
         }
-        this._parts.push(this._partCallback(this, part, walker.currentNode));
+        let node = walker.currentNode;
+        // Swap the comment placeholders for empty text nodes
+        // The nextSibling of the part placeholder is the part endNode
+        if (node.nextSibling !== null && node.nextSibling!.nodeType === 8) {
+          node.parentNode!.replaceChild(document.createTextNode(''), node.nextSibling!);
```

Comment:
Replacing each of these on every stamping seems unfortunate, but probably needs some perf gut checking. If this is an issue, it can be isolated to just where the issue occurs (IE11?)